### PR TITLE
Test find impulse response delay complex

### PR DESCRIPTION
**Pull request by tluebeck**
### Which issue(s) are closed by this pull request?

Closes #

### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: pyfar/classes/audio.py [hunk 1]
```diff
@@ -706,9 +706,9 @@ def __init__(
 
         # check fft norm
         if fft_norm in self._VALID_FFT_NORMS:
-            if self._complex and fft_norm == "rms":
-                raise ValueError(("'rms' normalization is not valid for "
-                                  "complex time signals"))
+            if self._complex and fft_norm in ["rms", "power"]:
```
Comment (ahms5) [inline pyfar/classes/audio.py:709]: how about ``'psd'``?

File: pyfar/classes/audio.py [hunk 2]
```diff
@@ -957,8 +957,8 @@ def fft_norm(self, value):
             raise ValueError(("Invalid FFT normalization. Has to be "
                               f"{', '.join(self._VALID_FFT_NORMS)}, but found "
                               f"'{value}'"))
-        if self._complex and value == "rms":
-            raise ValueError(("'rms' normalization is not valid for "
+        if self._complex and value in ["rms", "power"]:
```
Comment (ahms5) [inline pyfar/classes/audio.py:960]: how about ``'psd'``?

**Review by ahms5 — CHANGES_REQUESTED** [review]
I have just the confusion about the ``'psd'`` for normalization, otherwise its fine.

File: pyfar/classes/audio.py (hunk 1 above)
Comment (f-brinkmann) [reply pyfar/classes/audio.py:709]: This is part of #567 - should be fixe after that one is merged...

File: pyfar/classes/audio.py (hunk 2 above)
Comment (f-brinkmann) [reply pyfar/classes/audio.py:960]: see above

File: pyfar/dsp/dsp.py [hunk 3]
```diff
@@ -1248,54 +1255,65 @@ def find_impulse_response_delay(impulse_response, N=1):
...
+    modes = ['real', 'complex'] if impulse_response.complex else ['real']
+    start_sample = np.zeros((len(modes), 1), dtype=float)
+
     for ch in np.ndindex(impulse_response.cshape):
         # Calculate the correlation between the impulse response and its
         # minimum phase equivalent. This requires a minimum phase equivalent
         # in the strict sense, instead of the appriximation implemented in
         # pyfar.
         n_samples = impulse_response.n_samples
+        for idx, mode in zip(range(0, len(modes)), modes):
```
Comment (f-brinkmann) [inline pyfar/dsp/dsp.py:1267]: This looks better
```suggestion
        for idx, mode in enumerate(modes):
```

**Review by f-brinkmann — CHANGES_REQUESTED** [review]
Are all of those changes required for the delay estimation? There might be other changes in this PR.

File: pyfar/dsp/dsp.py (hunk 3 above)
Comment (tluebeck) [reply pyfar/dsp/dsp.py]: thanks, your're right.

Comment (tluebeck) [reply pyfar/dsp/dsp.py]: I think we need all these changes, see the discussion in slack:
We want to calculate the delay independently for real and imaginary part, and return the minimum of both. I don't see a more elegant way.

**Review by f-brinkmann — APPROVED** [review]

**Review by ahms5 — APPROVED** [review]

**Pull request merged**